src/libYARP_sig/include/yarp/sig/Image.h

```diff
@@ -589,6 +589,9 @@ class yarp::sig::ImageOf : public Image
     }
 };
 
+namespace yarp {
+namespace sig {
+
 template<>
 inline int yarp::sig::ImageOf<yarp::sig::PixelMono>::getPixelCode() const {
```

Comment:
Can you try if this still works?

```diff
-inline int yarp::sig::ImageOf<yarp::sig::PixelMono>::getPixelCode() const {
+inline int ImageOf<yarp::sig::PixelMono>::getPixelCode() const {
```

the double namespace looks wrong in this way, I'm afraid it could break on some compilers...

Reply:
Ops, sorry, I just read the following comment 🙇 
Can you please remove the `yarp::sig::` then?

Reply:
I changed my mind, I'll just merge this since all the test passed, and I'll open a new PR for the removal